### ISSUE TITLE
Any possibility to change the script to search for ARM images?

### ISSUE THREAD

**Issue by xeno-john** [pbatard/Fido#46]
The title pretty much says it all. I have stumbled across this repository and just wondered if the code can somehow be changed to also include ARM ISOs.

**Comment by pbatard** [comment]
Let me know when you see Microsoft offer ARM ISO downloads appear at https://www.microsoft.com/en-us/software-download.

As explained in the README, all Fido does is _"perform the same operation as one might perform when visiting the (above) URL"_, therefore, as long as Microsoft does not publish retail ARM64 ISOs there, Fido will never be able to let you download one.

Please don't confuse what this script does (which is allow the download of official __retail__ ISO, meaning ISOs that have been built once by Microsoft and set in stone, and that customers can actually purchase as DVDs in official Microsoft packaging in a store) with what sites like https://uupdump.net propose, which are __reconstructed ISOs__, that are built from picking a bunch of components here and there, and that have absolutely nothing official and are __NOT__ endorsed by Microsoft.

So while sites like uupdump.net make it look like one might be able to download ARM64 Windows ISOs, the reality is that Microsoft has not published any ARM64 retail Windows ISOs (which is evidenced by the fact that if you go to https://www.microsoft.com/en-US/software-download/windows11 the only architecture you'll be able to download is `x64`), which transitively mean that Fido cannot provide such ISOs to you.

In other words, if you come here with the idea that Fido should allow you to get the same ISOs as the ones you get from uupdump.net, you are misconstruing what this application is about.
